hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -40,6 +40,11 @@
     private String fullPath;
 
     /**
+     * Relative path to the file from the base path
+     */
+    private String relativePath;
```

Comment:
To also remove it from per file entries like below 

```
 "2015/03/17" : [ {
      "fileId" : "ba57976a-bee6-49ee-8e95-df9c04f0e02c",
      "fullPath" : "file:/tmp/hoodie/sample-table/2015/03/17/ba57976a-bee6-49ee-8e95-df9c04f0e02c_20170426135143.avro.delta.1",
      "prevCommit" : "20170426135143",
      "numWrites" : 33,
      "numDeletes" : 0,
      "numUpdateWrites" : 0,
      "totalWriteBytes" : 0,
      "totalWriteErrors" : 0,
      "logVersion" : 1,
      "logOffset" : 723
    } ]

```
I think that would be safe too.. @prazanna can you double confirm no-one else is reading the `fullPath` field above?  esp HoodieIncrementalPull - I think its running old code that filters commits based on whats in teh file name..